src/store.ts

```diff
@@ -1,25 +1,8 @@
-import {createStore, combineReducers, Unsubscribe} from "redux";
+import {createStore, combineReducers} from "redux";
 import {xChatReducers} from "~/modules/x-chat";
 
 export const store = createStore(
     combineReducers({
         ...xChatReducers,
     })
-);
-
-export function subscribeWithSelector(selector: Function, cb: Function): Unsubscribe {
```

Comment:
is it ok?

Reply:
it's not needed since it's in @micro-frontends/core package